scripts/node.sh

```diff
@@ -101,17 +108,25 @@ fi
 # find my public ip address
 myip
 
-# public beacon node multiaddress
-BC_MA=/ip4/54.183.5.66/tcp/9999/ipfs/QmdQVypu6NSm7m8bNZj5EJCnjPhXR8QyRmDnDBidxGaHWi
+# public boot node multiaddress
+BN_MA=/ip4/100.26.90.187/tcp/9876/p2p/QmZJJx6AdaoEkGLrYG4JeLCKeCKDjnFz2wfHNHxAqFSGA9,/ip4/54.213.43.194/tcp/9876/p2p/QmQayinFSgMMw5cSpDUiD9pQ2WeP6WNmGxpZ6ou3mdVFJX
```

Comment:
Good enough for now, but hardcoding these addresses/port numbers/node IDs is not such a good idea.  Opened #591 to track this.